src/include/daos_srv/daos_server.h

```diff
@@ -107,6 +107,24 @@ dss_tls_get()
 		pthread_getspecific(dss_tls_key);
 }
 
+#define D_TIME_START(sp, time) 				\
```

Comment:
(style)  please, no space before tabs

(style)  do {} while (0) macros should not be semicolon terminated